src/utils/utils.js

```diff
@@ -677,7 +677,7 @@ export function getDeepestPosition(node, offset) {
         let newNode = node.childNodes[offset];
         if (newNode) {
             newNode = getNextVisibleNode(newNode);
-            if (!newNode || isEmptyBlock(newNode)) break;
+            if (!newNode) break;
```

Comment:
I did not ask the question to the author but it does not break any test and comform better to the function intended behavior.
It was @Zinston that made the original line. @Zinston, do you remember for which case you needed `|| isEmptyBlock(newNode)` ?

Reply:
I hate to throw back the ball at someone else again but if you blame far enough you'll see this was actually written by @EmilienD ;-) Here's the PR that introduced that check that I simply made sure would still be done when I refactored: https://github.com/odoo-dev/odoo-editor/pull/144

Reply:
@Zinston Snitches get stitches :smile: 
@Goaman It's linked to `[SBU] UX POV: I expect the image to be removed on backspace: https://drive.google.com/file/d/1nXCGt9fLe7MugasvR_tziuCTJUAmmmum/view` The function itself has changed since the PR that introduced this, it might cover a scenario that is irrelevant now. You'll fix the bug yourself if it comes back I suppose.

Reply:
@dmo-odoo Indeed with my commit, the bug came back. I have added `isVisibleEmpty` which prevent the range going into img, br, and input.